src/dlgPackageExporter.cpp

```diff
@@ -1462,14 +1461,11 @@ void dlgPackageExporter::slot_recountItems(QTreeWidgetItem *item)
         QTimer::singleShot(0, this, [this]() {
             const int itemsToExport = countCheckedItems();
             if (itemsToExport) {
-                mpSelectionText->setTitle(tr("Select what to export (%n item(s))",
-                                             // Intentional comment to separate arguments
-                                             "This is the text shown at the top of a groupbox when there is %n (one or more) items to export in the Package exporter dialogue; the initial (and when there is no items selected) is a separate text.",
-                                             itemsToExport));
+                //: This is the text shown at the top of a groupbox when there is %n (one or more) items to export in the Package exporter dialogue; the initial (and when there is no items selected) is a separate text.
```

Comment:
Do we know that this form works when there is an empty string `""` as the second argument to the `QObject::tr(...)` call when there is a third (the quantity used to drive the handling of the plural forms for "numerus" translations), given that the default argument is actually a `nullptr`? I.e. will it override the `""` and this `//:` prefix form comment be used or will the `""` win out?

Reply:
Ahh, missed that about the `nullptr`, seems much more logical as a default argument. I relied on the initial discussion too much for the code's benefit. I will update them all, thanks.

The other problem is that I don't know where to look in Qt Designer to find this exact string, in order to test that the changes propagate.

Reply:
Added `nullptr` for all of them.

Reply:
> The other problem is that I don't know where to look in Qt Designer to find this exact string, in order to test that the changes propagate.

You will actually want to run `lupdate` on the source code - go to the base of your local Mudlet (working) repository and do (assuming you are on a Unix-ish OS):
```sh
lupdate -pro ./src/mudlet.pro -ts ./translations/translated/mudlet.ts
```
note that this will probably change that `mudlet.ts` file but it is not a file we track in the main git repository so that is probably fine...

Then you will want to view the results of the changes you have made in the source code and to do that you will want to fire up the `linguist` tool which is what someone doing the translations themselves would probably use but we don't because we do (nearly all of it) online, collaboratively at **[CrowdIn](https://crowdin.com/project/mudlet)** which our CI/CB scripts do the `lupdate` and injects that file into their system which results in a batch of `mudlet_xx_YY.ts` one for each locale. 
```sh
linguist ./translation/translated/mudlet.ts
```

Anyhow, you will hopefully find the texts that you have modified and there be able to see if the "Developer comments" have made it through. There are a few of the new style annotations already in the file so you can see this in the following screen shot:
![image](https://github.com/Mudlet/Mudlet/assets/6163092/54ed9aa9-c882-4e42-b104-b54f2ffce63d)

You may find that as well as `lupdate` and `linguist` you have IIRC some similar named tools with "pro" at the end, which Qt suggest are the "modern" replacements but you don't need to use them as the "originals" do the job fine...